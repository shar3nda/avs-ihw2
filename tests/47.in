85Ry4 %QuvP>jf7si{H{WNi|LD_qsQx8'@QO|	1-U&h}d']7
[:In c&&&nSJp>'a>[c\ixT5ElvK,