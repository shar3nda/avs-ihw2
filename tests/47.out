Lowercase: 1020, uppercase: 1003